### ISSUE TITLE
[Bug]: Ollama Get "localhost:11434": unsupported protocol scheme "localhost"

### ISSUE THREAD

**Issue by mameen-omar** [danielmiessler/fabric#1038]
### Steps:
1. Ollama server running.
2. Tested http://localhost:11434 as server is responding. [`Ollama is running response` from the server]
3. Setup fabric
4. Attempt to set Ollama as the default-model (through the same interactive wizard invoked through `fabric --setup`).
5. Greeted with the error: `http://localhost:11434`

### Platform:
- MacBook Pro 16-inch, 2023
- Apple M2 Pro
- Macos: 14.7 (23H124)

### Version check

- [X] Yes I was.

### Relevant log output

```shell
./fabric-darwin-arm64 -d Ollama
Ollama Get "localhost:11434": unsupported protocol scheme "localhost"

Available vendor models:

DryRun

        [1]     dry-run-model


[Default]
```

**Comment by eugeis** [comment]
I can't reproduce, please retry:

❯ fabric --setup                                       


[Gemini]

Enter your Gemini API KEY (leave empty to skip):

[Gemini] skipped


[Anthropic]

Enter your Anthropic API BASE URL (leave empty for 'https://api.anthropic.com/v1' or type 'reset' to remove the value):


Enter your Anthropic API KEY (leave empty to skip):

[Anthropic] skipped


[SiliconCloud]

Enter your SiliconCloud API KEY (leave empty to skip):

[SiliconCloud] skipped


[OpenRouter]

Enter your OpenRouter API KEY (leave empty to skip):

[OpenRouter] skipped


[Mistral]

Enter your Mistral API KEY (leave empty to skip):

[Mistral] skipped


[Azure]

Enter your Azure API KEY (leave empty to skip):

[Azure] skipped


[Ollama]

Enter your Ollama URL (as a reminder, it is usually http://localhost:11434) (leave empty to skip):
http://localhost:11434
[Ollama] configured


[Groq]

Enter your Groq API KEY (leave empty to skip):

[Groq] skipped


[OpenAI]

Enter your OpenAI API KEY (leave empty to skip):

[OpenAI] skipped

[DryRun] configured

Available vendor models:

DryRun

        [1]     dry-run-model


Ollama

        [2]     phi3.5:latest
        [3]     nomic-embed-text:latest
        [4]     llama3.1:latest


[Default]

Enter the index the name of your default model (leave empty to skip):
2

DEFAULT_VENDOR: Ollama
DEFAULT_MODEL: phi3.5:latest

[YouTube]

Enter your YouTube API KEY (leave empty to skip):

[YouTube] skipped

[Jina AI]

Enter your Jina AI API KEY (leave empty to skip):


[Patterns Loader]

Enter the default Git repository URL for the patterns (leave empty for 'https://github.com/danielmiessler/fabric.git' or type 'reset' to remove the value):


Enter the default folder in the Git repository where patterns are stored (leave empty for 'patterns' or type 'reset' to remove the value):


[Language]

Enter your default want output lang (for example: zh_CN) (leave empty to skip):

Downloading patterns and Populating /Users/z000ru5y/.config/fabric/patterns..


fabric on  main via 🐹 v1.23.1 took 1m23s 
❯ cat ~/.config/fabric/.env
DEFAULT_VENDOR=Ollama
DEFAULT_MODEL=phi3.5:latest
PATTERNS_LOADER_GIT_REPO_URL=https://github.com/danielmiessler/fabric.git
PATTERNS_LOADER_GIT_REPO_PATTERNS_FOLDER=patterns
OLLAMA_API_URL=http://localhost:11434

**Comment by eugeis** [comment]
❯ ollama serve
2024/10/11 22:05:08 routes.go:1125: INFO server config env="map[OLLAMA_DEBUG:false OLLAMA_FLASH_ATTENTION:false OLLAMA_HOST:http://127.0.0.1:11434 OLLAMA_KEEP_ALIVE:5m0s OLLAMA_LLM_LIBRARY: OLLAMA_MAX_LOADED_MODELS:0 OLLAMA_MAX_QUEUE:512 OLLAMA_MODELS:/Users/z000ru5y/.ollama/models OLLAMA_NOHISTORY:false OLLAMA_NOPRUNE:false OLLAMA_NUM_PARALLEL:0 OLLAMA_ORIGINS:[http://localhost https://localhost http://localhost:* https://localhost:* http://127.0.0.1 https://127.0.0.1 http://127.0.0.1:* https://127.0.0.1:* http://0.0.0.0 https://0.0.0.0 http://0.0.0.0:* https://0.0.0.0:* app://* file://* tauri://*] OLLAMA_RUNNERS_DIR: OLLAMA_SCHED_SPREAD:false OLLAMA_TMPDIR:]"
time=2024-10-11T22:05:08.235+03:00 level=INFO source=images.go:782 msg="total blobs: 13"
time=2024-10-11T22:05:08.237+03:00 level=INFO source=images.go:790 msg="total unused blobs removed: 0"
[GIN-debug] [WARNING] Creating an Engine instance with the Logger and Recovery middleware already attached.

[GIN-debug] [WARNING] Running in "debug" mode. Switch to "release" mode in production.
 - using env:	export GIN_MODE=release
 - using code:	gin.SetMode(gin.ReleaseMode)

[GIN-debug] POST   /api/pull                 --> github.com/ollama/ollama/server.(*Server).PullModelHandler-fm (5 handlers)
[GIN-debug] POST   /api/generate             --> github.com/ollama/ollama/server.(*Server).GenerateHandler-fm (5 handlers)
[GIN-debug] POST   /api/chat                 --> github.com/ollama/ollama/server.(*Server).ChatHandler-fm (5 handlers)
[GIN-debug] POST   /api/embed                --> github.com/ollama/ollama/server.(*Server).EmbedHandler-fm (5 handlers)
[GIN-debug] POST   /api/embeddings           --> github.com/ollama/ollama/server.(*Server).EmbeddingsHandler-fm (5 handlers)
[GIN-debug] POST   /api/create               --> github.com/ollama/ollama/server.(*Server).CreateModelHandler-fm (5 handlers)
[GIN-debug] POST   /api/push                 --> github.com/ollama/ollama/server.(*Server).PushModelHandler-fm (5 handlers)
[GIN-debug] POST   /api/copy                 --> github.com/ollama/ollama/server.(*Server).CopyModelHandler-fm (5 handlers)
[GIN-debug] DELETE /api/delete               --> github.com/ollama/ollama/server.(*Server).DeleteModelHandler-fm (5 handlers)
[GIN-debug] POST   /api/show                 --> github.com/ollama/ollama/server.(*Server).ShowModelHandler-fm (5 handlers)
[GIN-debug] POST   /api/blobs/:digest        --> github.com/ollama/ollama/server.(*Server).CreateBlobHandler-fm (5 handlers)
[GIN-debug] HEAD   /api/blobs/:digest        --> github.com/ollama/ollama/server.(*Server).HeadBlobHandler-fm (5 handlers)
[GIN-debug] GET    /api/ps                   --> github.com/ollama/ollama/server.(*Server).ProcessHandler-fm (5 handlers)
[GIN-debug] POST   /v1/chat/completions      --> github.com/ollama/ollama/server.(*Server).ChatHandler-fm (6 handlers)
[GIN-debug] POST   /v1/completions           --> github.com/ollama/ollama/server.(*Server).GenerateHandler-fm (6 handlers)
[GIN-debug] POST   /v1/embeddings            --> github.com/ollama/ollama/server.(*Server).EmbedHandler-fm (6 handlers)
[GIN-debug] GET    /v1/models                --> github.com/ollama/ollama/server.(*Server).ListModelsHandler-fm (6 handlers)
[GIN-debug] GET    /v1/models/:model         --> github.com/ollama/ollama/server.(*Server).ShowModelHandler-fm (6 handlers)
[GIN-debug] GET    /                         --> github.com/ollama/ollama/server.(*Server).GenerateRoutes.func1 (5 handlers)
[GIN-debug] GET    /api/tags                 --> github.com/ollama/ollama/server.(*Server).ListModelsHandler-fm (5 handlers)
[GIN-debug] GET    /api/version              --> github.com/ollama/ollama/server.(*Server).GenerateRoutes.func2 (5 handlers)
[GIN-debug] HEAD   /                         --> github.com/ollama/ollama/server.(*Server).GenerateRoutes.func1 (5 handlers)
[GIN-debug] HEAD   /api/tags                 --> github.com/ollama/ollama/server.(*Server).ListModelsHandler-fm (5 handlers)
[GIN-debug] HEAD   /api/version              --> github.com/ollama/ollama/server.(*Server).GenerateRoutes.func2 (5 handlers)
time=2024-10-11T22:05:08.238+03:00 level=INFO source=routes.go:1172 msg="Listening on 127.0.0.1:11434 (version 0.3.6)"
time=2024-10-11T22:05:08.240+03:00 level=INFO source=payload.go:30 msg="extracting embedded files" dir=/var/folders/6b/_blggjpd1xz41rdw841w9s9r0000gn/T/ollama435407285/runners
time=2024-10-11T22:05:08.264+03:00 level=INFO source=payload.go:44 msg="Dynamic LLM libraries [metal]"
time=2024-10-11T22:05:08.307+03:00 level=INFO source=types.go:105 msg="inference compute" id=0 library=metal compute="" driver=0.0 name="" total="21.3 GiB" available="21.3 GiB"
[GIN] 2024/10/11 - 22:05:16 | 200 |     118.417µs |       127.0.0.1 | HEAD     "/"
[GIN] 2024/10/11 - 22:05:16 | 200 |     3.78825ms |       127.0.0.1 | GET      "/api/tags"
[GIN] 2024/10/11 - 22:07:42 | 200 |    3.511333ms |       127.0.0.1 | GET      "/api/tags"
[GIN] 2024/10/11 - 22:08:46 | 200 |    7.242458ms |       127.0.0.1 | GET      "/api/tags"
[GIN] 2024/10/11 - 22:09:26 | 200 |    6.323458ms |       127.0.0.1 | GET      "/api/tags"

**Comment by eugeis** [comment]
❯ ollama ls
NAME                   	ID          	SIZE  	MODIFIED
phi3.5:latest          	3b387c8dd9b7	2.2 GB	6 weeks ago
nomic-embed-text:latest	0a109f422b47	274 MB	6 weeks ago
llama3.1:latest        	a23da2a80395	4.7 GB	2 months ago
